inc/Mailer.class.php

```diff
@@ -716,7 +717,14 @@ public function send() {
                 $headers .= MAILHEADER_EOL.trim($data['headers']);
             }
 
-            // send the thing
+            if(!function_exists('mail')){
+                $emsg = $lang['email_fail'] . $subject;
+                error_log($emsg);
+                msg($emsg,2,"","", MSG_MANAGERS_ONLY);
```

Comment:
Not sure how notify looks like, but -1 (error) should show the message box with a red background which is more suitable.

```suggestion
                msg($emsg,-1,"","", MSG_MANAGERS_ONLY);
```